Authentication/Login/View/LoginController.swift

```diff
@@ -12,13 +12,19 @@ import Rex
 import enum Result.NoError
 
 
-/**
-    Log In View Controller that takes care of managing the login, from
+/*
+    Login View Controller that takes care of managing the login, from
     validating email and password fields, to binding a login view to the
-    view model and informing the log in controller delegate when certain
+    view model and informing the login controller delegate when certain
     events occur for it to act upon them.
-    If there are more than one errors in a field, the controller presents
-    only the first one in the errors label.
+    If there are more than one validation error in a field, the controller
+    presents only the first one in the errors label.
+ 
+    If wanting to use the default LoginController with some customization,
+    you should not override the `createLoginController` method of the Bootstrapper,
```

Comment:
Also I don't think this should be explained here because all those methods are part of the bootstrapper. I think that the documentation should only "talk" about the class being documented not how other classes use it.